client/signup/steps/intent/intent-screen.tsx

```diff
@@ -35,6 +37,14 @@ const useIntents = ( { translate } ): Intent[] => {
 			intent: 'build',
 			actionText: translate( 'Start building' ),
 		},
+		{
+			title: '',
+			description: 'Let our experts create your dream site',
+			icon: null,
+			intent: 'do-it-for-me',
+			actionText: translate( 'Do it for me' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Do It For Me (Lite)' )` *ES Score: 7*